tests/validation/tests/v3_api/common.py

```diff
@@ -1026,6 +1026,13 @@ def validate_dns_entry(pod, host, expected, port=TEST_IMAGE_PORT):
         cmd = 'ping -c 1 -W 1 {0}'.format(host)
     cmd_output = kubectl_pod_exec(pod, cmd)
 
+    for i in range(0, retry_count):
+        cmd_output = kubectl_pod_exec(pod, cmd)
+        if str(pod.name) in str(cmd_output):
```

Comment:
Could we add this validation before entering the `for` loop so we don't retry on an already successful output.

Reply:
I've added a logic. Please review again.